django/test/utils.py

```diff
@@ -186,8 +186,7 @@ def setup_databases(verbosity, interactive, keepdb=False, debug_sql=False, paral
 
     # Configure the test mirrors.
     for alias, mirror_alias in mirrored_aliases.items():
-        connections[alias].creation.set_as_test_mirror(
-            connections[mirror_alias].settings_dict)
+        connections[alias] = connections[mirror_alias]
```

Comment:
Is this right? Isn't the intention here to set the `mirror_alias` connection to the `alias` connection?

Reply:
The `mirrored_aliases` actually contains database -> mirror. So if we have database `mirror` that mirrors `default` the code above results in:

`connections['mirror'] = connections['default']`